prow/cmd/branchprotector/protect.go

```diff
@@ -63,45 +62,22 @@ func gatherOptions() options {
 	return o
 }
 
-func jobRequirements(jobs []config.Presubmit, after bool) []string {
-	var required []string
-	for _, j := range jobs {
-		// Does this job require a context or have kids that might need one?
-		if !after && !j.AlwaysRun && j.RunIfChanged == "" {
-			continue // No
-		}
-		if !j.SkipReport { // This job needs a context
```

Comment:
One thing we need is to change this to `if !j.SkipReport && !j.Optional`

Reply:
what is optional ?